Node_Editor/Framework/NodeEditor.cs

```diff
@@ -265,7 +268,23 @@ public static Node NodeAtPosition (NodeEditorState editorState, Vector2 canvasPo
 						return node;
 					}
 				}
-			}
+                for (int knobCnt = 0; knobCnt < node.Inputs.Count; knobCnt++)
+                { // Check if any nodeKnob is focused instead
+                    if (node.Inputs[knobCnt].GetCanvasSpaceKnob().Contains(canvasPos))
+                    {
+                        focusedKnob = node.Inputs[knobCnt];
+                        return node;
+                    }
+                }
+                for (int knobCnt = 0; knobCnt < node.Outputs.Count; knobCnt++)
+                { // Check if any nodeKnob is focused instead
+                    if (node.Outputs[knobCnt].GetCanvasSpaceKnob().Contains(canvasPos))
+                    {
+                        focusedKnob = node.Outputs[knobCnt];
+                        return node;
+                    }
+                }
+            }
```

Comment:
Yes, its the code i forgot to comment, now that i think about this, i will remove all the code that i commented for my previous implementation of removing the nodeknob list